src/components/app/details/triggerView/cdMaterial.tsx

```diff
@@ -86,6 +88,7 @@ export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
             latestDeploymentConfig: null,
             specificDeploymentConfig: null,
             isSelectImageTrigger: props.materialType === MATERIAL_TYPE.inputMaterialList,
+            isEditMode: new Map<number,boolean>(),
```

Comment:
Please rename the variable to relatable context

Reply:
done